qadence/mitigations/readout.py

```diff
@@ -89,5 +130,9 @@ def mitigation_minimization(
     return corrected_counters
 
 
-def mitigate(noise: Noise, mitigation: Mitigations, samples: list[Counter]) -> list[Counter]:
-    return mitigation_minimization(noise=noise, mitigation=mitigation, samples=samples)
+def mitigate(
+    noise: Noise, mitigation: Mitigations, samples: list[Counter], optimization_type: str = "mle"
```

Comment:
OK, a possible way to keep a different option for calculating the matrix, is to add the optimization_type to the Mitigations.READOUT options. Then, the mitigation_minimization should be able to use the information from the opt dict.